### PR TITLE
Implement WebAuthn basic test using testdriver API

### PR DESCRIPTION
**Pull request by nsatragno**
This change adds a test for successfully creating a credential and
getting an assertion, automated using the testdriver.js WebAuthn API.

### CODE REVIEW

**Comment by nsatragno** [comment]
@jcjones please take a look.

What do you think about having `add_virtual_authenticator` not do anything if the browser running the test doesn't support the API? That way we could add code to automate the existing WPTs without breaking manual testing.

**Review by jcjones — APPROVED** [review]
Approved assuming the licensing is OK!

File: webauthn/resources/common-inputs.js
Comment (nsatragno) [reply webauthn/resources/common-inputs.js]: Done.

File: webauthn/resources/common-inputs.js
Comment (nsatragno) [reply webauthn/resources/common-inputs.js]: Extracted into a constant on this file.

File: webauthn/resources/utils.js
Comment (nsatragno) [reply webauthn/resources/utils.js]: I asked and it's fine :)

File: webauthn/webauthn-testdriver-basic.https.html
Comment (nsatragno) [reply webauthn/webauthn-testdriver-basic.https.html]: Yeah, it's intentional :( There's a `resources` folder in the `webauthn` folder with these scripts...

**Review by nsatragno — COMMENTED** [review]
Comments addressed:

**Comment by nsatragno** [comment]
FYI this is ready to merge @jcjones :smiley:  (I don't have write access to this repo)

**Review by bdstar81 — APPROVED** [review]

**Pull request merged**
